GirafIntegrationTest/tests/test_user_controller.py

```diff
@@ -261,11 +272,12 @@ def test_user_can_get_citizen3_id(self):
         Endpoint: GET:/v1/User
         """
         global citizen3_id
-        response = get(f'{BASE_URL}v1/User', headers=auth(citizen3_token)).json()
-        self.assertTrue(response['success'])
-        self.assertEqual(response['errorKey'], 'NoError')
-        self.assertIsNotNone(response['data']['id'])
-        citizen3_id = response['data']['id']
+        response = get(f'{BASE_URL}v1/User', headers=auth(citizen3_token))
+        response_body = response.json()
+        self.assertEqual(response.status_code, HTTPStatus.OK)
+
+        self.assertIsNotNone(response_body['data']['id'])
```

Comment:
This will raise a `KeyError` if the request fails, consider using [`dict.get()`](https://docs.python.org/3/library/stdtypes.html?highlight=dict%20get#dict.get) instead - or check on `'data'` first.